src/css/components/_material-switch.scss

```diff
@@ -5,69 +5,67 @@
   position: relative;
   margin-right: 19px;
 
-  & .track {
+  .track {
     border-radius: 19px;
     position: absolute;
     top: 0;
     bottom: 0;
     left: 1px;
     right: 1px;
-    background: rgba(#000, 0.26);
+    background-color: rgba(#000, .26);
     overflow: hidden;
     transform: translateZ(0);
 
     &::before {
-      content: '';
+      content: "";
       position: absolute;
       top: 0;
       left: 0;
       right: 0;
       bottom: 0;
-      background: #9FA8DA;
+      background-color: #9fa8da;
       opacity: 0;
-      transition: opacity 0.15s ease-out;
+      transition: opacity .15s ease-out;
       transform: translateZ(0);
     }
   }
 
-  & .handle {
+  .handle {
     position: absolute;
     top: -3px;
     left: 0;
     width: 20px;
     height: 20px;
     border-radius: 20px;
-    background: #fafafa;
-    box-shadow: 0 1px 3px rgba(0,0,0,0.5), 
-                inset 0 0 0 2px transparent;
+    background-color: #fafafa;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .5), inset 0 0 0 2px transparent;
```

Comment:
BTW I wouldn't mind keeping these but I couldn't figure out the proper stylelint combination. If someone else figures it out, let me know.